ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -96,10 +96,7 @@ public static JsonDecoder createJsonDecoder() {
 
   private HttpService createWeb3jHttpService() {
     final String downstreamUrl =
-        "http://"
-            + config.getDownstreamHttpHost().getHostName()
-            + ":"
-            + config.getDownstreamHttpPort();
+        "http://" + config.getDownstreamHttpHost() + ":" + config.getDownstreamHttpPort();
```

Comment:
may be take "scheme" out in case we switch to https in future?

Reply:
okhttp which is used by web3j requires a scheme to be provided